department-of-veterans-affairs/va.gov-cms
# VACMS-4581: Require owner field on vc health service.

## Description
See #4581

## Testing done
Visual / Behat

## QA steps
- As an administrator, go to `admin/structure/types/manage/vet_center_facility_health_servi/fields/node.vet_center_facility_health_servi.field_administration` and visually verify the required box is checked.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
